content/en/case-studies/gw-discov.md

```diff
@@ -3,49 +3,70 @@ title: "Case Study: Discovery of Gravitational Waves"
+universe such as collision and merging of two black holes, coalescing binary
+stars, supernovae or [compact
+binaries](https://wwwmpa.mpa-garching.mpg.de/~hsr/researchcb-en.html). Observing
+GW can not only help in studying gravity but also in understanding some of the
+obscure phenomena in the distant universe and its impact. Two of the most recent
+events in the context of gravitaional waves are the focus of this case study:
+
+* [GW observations from LIGO/Virgo
+GW150914](https://en.wikipedia.org/wiki/First_observation_of_gravitational_waves):
+The original discovery of gravitational waves from a binary black hole merger
+* [GW observations from LiGO/Virgo
+GW170817](https://en.wikipedia.org/wiki/GW170817): The first direct detection
+of neutron start mergers 
+
+The [Laser Interferometer Gravitational-Wave Observatory
+(LIGO)](https://www.ligo.caltech.edu) was designed to open the field of
+gravitational-wave astrophysics through the direct detection of gravitational
+waves predicted by Einstein’s General Theory of Relativity. It comprises two
+widely-separated interferometers within the United States—one in Hanford,
+Washington and the other in Livingston, Louisiana—operated in unison to detect
+gravitational waves. Each of them has multi-kilometer-scale gravitational wave
+detectors that use laser interferometry.  The LIGO Scientific Collaboration
+(LSC), is a group of more than 1000 scientists from universities around the
+United States and in 14 other countries supported by more than 90 universities
+and research institutes; approximately 250 students actively contributing to the
+collaboration. LIGO discovery GW150914 is the first observation that is
+significant in being the first observation both confirming the existence of
+gravitational waves as well as proves the ability to directly measure them. 
+
+Gravitational waves allow scientists to observe and study the universe in a
+brand new way, ushering in the era of gravitational wave astronomy.
 
 ### Key Objectives
 
 * Though its [mission](https://www.ligo.caltech.edu/page/what-is-ligo) is to
   detect gravitational waves from some of the most violent and energetic
-  processes in the Universe, the data LIGO collects may have far-reaching
-  effects on many areas of physics including gravitation, relativity,
-  astrophysics, cosmology, particle physics, and nuclear physics.
+processes in the Universe, the data LIGO collects may have far-reaching effects
+on many areas of physics including gravitation, relativity, astrophysics,
+cosmology, particle physics, and nuclear physics.
 * Crunch observed data via numerical relativity computations that involves
   complex maths in order to discern signal from noise, filter out relevant
-  signal and statistically estimate significance of observed data 
+signal and statistically estimate significance of observed data 
 * Data visualization so that the binary / numerical results can be
   comprehended.
```

Comment:
```suggestion
* Data visualization so that the numerical results can be
  comprehended.
```
I'm not sure about this bullet. Visualization is nice, but I wouldn't count is as a key objective, nor is the main purpose necessarily comprehension. Perhaps rephrasing to emphasize the generation of high-level results suitable for comparing with model predictions and visualization.